docs/contracts/functions.rst

```diff
@@ -297,7 +297,7 @@ on plain Ether transfers (e.g. via ``.send()`` or ``.transfer()``). If no such
 function exists, but a payable :ref:`fallback function <fallback-function>`
 exists, the fallback function will be called on a plain Ether transfer. If
 neither a receive Ether nor a payable fallback function is present, the
-contract cannot receive Ether through regular transactions and throws an
+contract cannot receive Ether through transactions that do not match a payable function and throws an
```

Comment:
I'd adjust the wording a bit. It might not be clear what it means for the whole transaction to "match". We usually say that referring specifically to a selector.
```suggestion
contract cannot receive Ether through a transaction that does not represent a payable function call and throws an
```